{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/pytest-dev/pytest-faulthandler",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-1.6.0.tar.gz",
    "subdirs": [
      "linux-ppc64le"
    ],
    "summary": "py.test plugin that activates the fault handler module for tests",
    "tags": null,
    "text_prefix": false,
    "timestamp": 1560568115,
    "version": "1.6.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main"
  ],
  "package": "pytest-faulthandler-1.6.0-py37_0.tar.bz2",
  "repodata": {
    "build": "py37_0",
    "build_number": 0,
    "depends": [
      "pytest",
      "python >=3.7,<3.8.0a0"
    ],
    "license": "MIT",
    "md5": "0021fb5ba6ad197cd958f8f4a1a05924",
    "name": "pytest-faulthandler",
    "sha256": "9f6278de9f0ad412811dcc5ec9e432ae41d6703fb829b31bf50d54fde58b586f",
    "size": 7985,
    "subdir": "linux-ppc64le",
    "timestamp": 1560568115556,
    "version": "1.6.0"
  },
  "repodata_version": 1,
  "subdir": "linux-ppc64le",
  "url": "https://conda.anaconda.org/conda-forge/linux-ppc64le/pytest-faulthandler-1.6.0-py37_0.tar.bz2"
}